From 0e5abaabb6188fd5247e711199b60cc5af8c94c4 Mon Sep 17 00:00:00 2001
From: Szilard Nemeth <snemeth@cloudera.com>
Date: Sat, 11 Aug 2018 07:00:35 +0200
Subject: [PATCH 2824/2848] CDH-71644. CLOUDERA-BUILD. Use maven settings.xml
 and GRIND_MAVEN_FLAGS provided by the Jenkins job

==C5_APPROVED_BUGFIX==

Change-Id: Ic2f65332aa594c78f542c292156abc2901e23b2e
---
 cloudera/test-distributed.sh |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/cloudera/test-distributed.sh b/cloudera/test-distributed.sh
index 0c0b6d3..3ba9c93 100755
--- a/cloudera/test-distributed.sh
+++ b/cloudera/test-distributed.sh
@@ -32,6 +32,11 @@ if [[ -z $DIST_TEST_USER || -z $DIST_TEST_PASSWORD ]]; then
     source dist_test_cred.sh
 fi
 
+if [[ ! -z $DIST_TEST_MVN_SETTINGS_FILE ]]; then
+    echo "Using maven settings file from: $DIST_TEST_MVN_SETTINGS_FILE"
+    echo "maven_settings_file = $DIST_TEST_MVN_SETTINGS_FILE" >> ./env/grind.cfg
+fi
+
 # Go to project root
 cd "$DIR/.."
 
@@ -45,7 +50,6 @@ artifact_archive_globs = ["**/surefire-reports/TEST-*.xml"]
 EOF
 
 export DIST_TEST_URL_TIMEOUT=180
-export GRIND_MAVEN_FLAGS="${GRIND_MAVEN_FLAGS} -Dhttps.protocols=TLSv1.2"
 # Invoke grind to run tests
 grind -c ${DIR}/$SCRIPTS/env/grind.cfg config
 grind -c ${DIR}/$SCRIPTS/env/grind.cfg pconfig
@@ -56,7 +60,8 @@ grind -c ${DIR}/$SCRIPTS/env/grind.cfg test --artifacts -r 3 \
     -e TestContainerManagerSecurity \
     -e TestMRIntermediateDataEncryption \
     -e TestClientRMTokens \
-    -e TestAMAuthorization
+    -e TestAMAuthorization \
+    -e TestSSLHttpServer
     # TestClientRMTokens and TestAMAuthorization to be fixed in 5.8 (CDH-39590)
     # TestContinuousScheduling has been failing consistently, to be fixed in 5.8 (CDH-38830)
 
-- 
1.7.9.5

